src/cli/peakdetector/PeakDetectorCLI.h

```diff
@@ -52,6 +54,14 @@ class PeakDetectorCLI {
 	public:
 		PeakDetectorCLI();
 
+		QString username;
```

Comment:
@Ky5595 Lots of these member variables should be just local to some function and rather avoid them to be part of the class.

Reply:
@sahil21 I will add it to my TODO list and take up in next sprint

Reply:
@Ky5595 Can you add TODO(s) as code comments? Personal to-do lists have the ability to get lost :p

Reply:
@sahil21  Yes, Did that only.. You can find TODO(s) in the updated version of this PR..

Reply:
@Ky5595 Thanks!